README.md

```diff
@@ -43,6 +43,11 @@ You can find a great example of such an issue [here](https://github.com/nodejs/c
 
 Community Committee meetings will be broadcast via Zoom, will be announced ahead of time for access, and coordinated to optimize for contributor timezones.
 
+Meeting cadence is every other week on Thursdays.  Please visit the [issues](https://github.com/nodejs/community-committee/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+%22Node.js+Foundation+Community+Committee+Meeting%22) section for next scheduled meeting.  If none found, please feel free to ping any of the [members](#community-committee-members).
+
+FYI: The publicly open segment of the meetings are multi-cast on [YouTube](https://www.youtube.com/c/nodejs+foundation/live).  Just navigate there at meeting start time.  Please be aware that other meetings might be occurring right before.
```

Comment:
> Just navigate there at meeting start time.

I'd suggest striking this, as we usually don't start immediately on time to ensure that everyone can get in.

Also there are a couple spaces here with two spaces after the period rather than one.

Reply:
Hey, Tierney, I would venture to say that it's a better alternative be to *start on time* (exceptions aside of course), rather than not letting newcomers know what to expect.  Thoughts?

Also, two spaces in the standard number after periods for essays, etc.  But, I can adhere to rules here if necessary.

Reply:
While I agree starting on time is what we'd like to do, we've been doing this quite a while and the reality is that we always start late (best case is usuallty 5 mins). This is the wording used in the TSC meeting issues:
```
We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at https://www.youtube.com/c/nodejs+foundation/live when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.
```

Reply:
I like that wording!  Will add it in there!  Thx!  👍🏼

Reply:
@mhdawson that's a 👍 wording. @cdnadmin do you think you could adapt that?

Reply:
Also the majority of our documentation has single spaces. Mostly just asking for consistency above all else 👍

Reply:
I replaced my original wording with the quote from @mhdawson 

All should be good now.